From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 23 Jan 2017 15:22:56 +0000
Subject: cfg80211: avoid ABI change in 4.9.3
Forwarded: not-needed

enum nl80211_attrs got a new enumerator, which should be safe (it's
meant to be extensible).  But two exported functions take it as a
parameter type.  Hide the change from genksyms so it doesn't consider
this to change their ABI.

---
--- a/include/uapi/linux/nl80211.h
+++ b/include/uapi/linux/nl80211.h
@@ -2339,7 +2339,9 @@ enum nl80211_attrs {
 	NL80211_ATTR_NAN_FUNC,
 	NL80211_ATTR_NAN_MATCH,
 
+#if !defined(__KERNEL__) || !defined(__GENKSYMS__)
 	NL80211_ATTR_BSSID,
+#endif
 
 	/* add attributes here, update the policy in nl80211.c */
 
